honeybadgerbft/core/honeybadger.py

```diff
@@ -44,7 +44,7 @@ def broadcast_receiver_loop(recv_func, recv_queues):
 
 
 class HoneyBadgerBFT():
-    """HoneyBadgerBFT object used to run the protocol.
+    r"""HoneyBadgerBFT object used to run the protocol.
```

Comment:
Is this `r` a typo?

Reply:
No, it is for making the docstring a raw string so that backslashes for latex math are treated as literal characters. Otherwise `flake8` flags these as errors. :)